Tests/ListRemoveStringsMethodsTests.cs

```diff
@@ -6,6 +6,7 @@ namespace Tests
 {
     [TestFixture(typeof(MyArrayList<string>))]
     [TestFixture(typeof(MyLinkedList<string>))]
+    [TestFixture(typeof(MyDoublyLinkedList<string>))]
```

Comment:
Add some complex tests:
1) remove some element.
2) Then remove again
3) then compare collection
And remove+insert+remove+replacement + check